src/components/ConfigAnalysisLink/ConfigAnalysisLink.tsx

```diff
@@ -0,0 +1,168 @@
+import { useState, useEffect } from "react";
+import ReactTooltip from "react-tooltip";
+import { EvidenceType } from "../../api/services/evidence";
+import { formatISODate, formatLongDate } from "../../utils/date";
+import { AttachEvidenceDialog } from "../AttachEvidenceDialog";
+import { ConfigTypeInsights } from "../ConfigInsights";
+import ConfigInsightsIcon from "../ConfigInsightsIcon";
+import { Description } from "../Description/description";
+import { Icon, Avatar } from "../Icon";
+import { Modal } from "../Modal";
+
+type Props = {
+  configAnalysis: ConfigTypeInsights;
+  viewType?: "summary" | "detailed";
+} & React.HTMLProps<HTMLDivElement>;
+
+export function ConfigAnalysisLink({
+  configAnalysis,
+  viewType = "summary",
+  ...rest
+}: Props) {
+  const [attachEvidence, setAttachEvidence] = useState(false);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    ReactTooltip.rebuild();
+  });
+
+  return (
+    <div
+      className="flex flex-col cursor-pointer  space-y-1"
+      {...rest}
+      onClick={() => {
+        setOpen(true);
+      }}
+    >
+      <Modal
+        title={
+          <>
+            <ConfigInsightsIcon analysis={configAnalysis} />
+            {configAnalysis.analyzer}
+          </>
+        }
+        open={open}
+        onClose={(e) => {
+          e?.stopPropagation();
+          setOpen(false);
+        }}
+        size="large"
+        bodyClass=""
+      >
+        {configAnalysis?.id && (
+          <>
+            <div className="flex flex-col px-4 py-4 text-gray-500 space-y-2">
+              <div className="flex flex-col space-y-6">
+                <div className="grid grid-cols-1 gap-x-4 gap-y-8 sm:grid-cols-4">
+                  <div className="sm:col-span-1">
```

Comment:
This should be refactored into DescriptionCard with labelStyle="top", there shouldn't be any styling at this level it should all be in DescriptionCard - this ensures that all descriptions are styled consistently throughout - e.g. Config analysis and changes descriptions now have different fonts